bool setSearchedROI
bool searchedROIState
uint32 numSearchedROI
bool addNewROI
ROI newROI
---
